eli5/formatters/html.py

```diff
@@ -39,8 +43,10 @@ def format_as_html(explanation, include_styles=True, force_weights=True,
     in the document.
     If ``highlight_spaces`` is None (default), spaces will be highlighted in
     feature names only if there are any spaces at the start or at the end of the
-    feature. Setting it to True forces space highlighting, and setting it to False
-    turns it off.
+    feature. Setting it to True forces space highlighting, and setting it to
+    False turns it off.
+    ``dense_multitarget`` set to True displays a multitarget classifier weights
+    in a single table, and shows highlighted document only for winning target.
```

Comment:
How does it work for regression tasks? I'm not sure it makes sense to choose a 'winner' in multitarget regression.